src/app/unos-podataka/unos-podataka.component.ts

```diff
@@ -12,34 +13,152 @@ export class UnosPodatakaComponent implements OnInit {
+              [{25:"3,54"},{30:"4,69"},{35:"5,79"},{40:"6,89"},{45:"8,03"},{50:"9,16"},{55:"10,3"},{60:"11,42"},{65:"12,49"},{70:"13,54"}]],
 
+  ['cetvrti', [{25:"3,54"},{30:"4,69"},{35:"5,79"},{40:"6,89"},{45:"8,03"},{50:"9,16"},{55:"10,3"},{60:"11,42"},{65:"12,49"},{70:"13,54"}], 
+              [{25:"3,54"},{30:"4,69"},{35:"5,79"},{40:"6,89"},{45:"8,03"},{50:"9,16"},{55:"10,3"},{60:"11,42"},{65:"12,49"},{70:"13,54"}],
+              [{25:"3,54"},{30:"4,69"},{35:"5,79"},{40:"6,89"},{45:"8,03"},{50:"9,16"},{55:"10,3"},{60:"11,42"},{65:"12,49"},{70:"13,54"}], 
+              [{25:"3,54"},{30:"4,69"},{35:"5,79"},{40:"6,89"},{45:"8,03"},{50:"9,16"},{55:"10,3"},{60:"11,42"},{65:"12,49"},{70:"13,54"}],
+              [{25:"3,54"},{30:"4,69"},{35:"5,79"},{40:"6,89"},{45:"8,03"},{50:"9,16"},{55:"10,3"},{60:"11,42"},{65:"12,49"},{70:"13,54"}]],
+
+  ['peti',    [{25:"3,54"},{30:"4,69"},{35:"5,79"},{40:"6,89"},{45:"8,03"},{50:"9,16"},{55:"10,3"},{60:"11,42"},{65:"12,49"},{70:"13,54"}], 
+              [{25:"3,54"},{30:"4,69"},{35:"5,79"},{40:"6,89"},{45:"8,03"},{50:"9,16"},{55:"10,3"},{60:"11,42"},{65:"12,49"},{70:"13,54"}],
+              [{25:"3,54"},{30:"4,69"},{35:"5,79"},{40:"6,89"},{45:"8,03"},{50:"9,16"},{55:"10,3"},{60:"11,42"},{65:"12,49"},{70:"13,54"}], 
+              [{25:"3,54"},{30:"4,69"},{35:"5,79"},{40:"6,89"},{45:"8,03"},{50:"9,16"},{55:"10,3"},{60:"11,42"},{65:"12,49"},{70:"13,54"}],
+              [{25:"3,54"},{30:"4,69"},{35:"5,79"},{40:"6,89"},{45:"8,03"},{50:"9,16"},{55:"10,3"},{60:"11,42"},{65:"12,49"},{70:"13,54"}]],
+];
+
+bonitetCetinari;
+bonitetLiscari;
+usloviRada;
+bodoviCetinari;
+bodoviLiscari;
+y;
+z: number;
+nagibTerena;
+gustinaPodmlatka;
+doznacenaMasa;
+nadmorskaVisina;
+udaljenostOdCeste;
+srednjiPrecnikCetinara;
+srednjiPrecnikLiscari;
+
+onKey(event: any) {
+  this.srednjiPrecnikCetinara = event.target.value;
+}
 onChangeValue1(event){
-  this.nadmorskaVisina = event.target.value;
+  this.nagibTerena = event.target.value;
+  console.log(this.nagibTerena)
  }
+
 onChangeValue2(event){
+  this.gustinaPodmlatka = event.target.value;
+  console.log(this.gustinaPodmlatka)
+ }
+onChangeValue3(event){
+  this.doznacenaMasa = event.target.value;
+  console.log(this.doznacenaMasa)
+ }
+onChangeValue4(event){
+  this.nadmorskaVisina = event.target.value;
+  console.log(this.nadmorskaVisina)
+ }
+onChangeValue5(event){
+  this.udaljenostOdCeste = event.target.value;
+  console.log(this.udaljenostOdCeste)
+ }
+onChangeValue6(event){
    this.srednjiPrecnikCetinara = event.target.value;
+   console.log(this.srednjiPrecnikCetinara)
   }
+onChangeValue7(event){
+    this.srednjiPrecnikLiscari = event.target.value;
+    console.log(this.srednjiPrecnikLiscari)
+   }
+onChangeValue8(event){
+    this.bonitetCetinari = event.target.value;
+    console.log(this.bonitetCetinari)
+   }
+   onChangeValue9(event){
+    this.bonitetLiscari = event.target.value;
+    console.log(this.bonitetLiscari)
+   }
```

Comment:
sve ove console.logs iskljuci kad trazis merge, jer to je samo za tvoj proces rada i testiranja :)